tests/zfs-tests/tests/functional/redacted_send/redacted.kshlib

```diff
@@ -237,15 +237,11 @@ function redacted_cleanup
 	typeset ds_list=$@
 	typeset ds
 
-	# Verify the receiving pool can still be exported and imported.
-	log_must zpool export $POOL2
-	log_must zpool import $POOL2
```

Comment:
Was this meant to be deleted?

Reply:
Yes. I don't think we should consider cleanup functions to be an appropriate place for adding tests which potentially fail and cause us to move on to the next test without finishing the cleanup.

There are already tests in redacted_send that perform import and export, so I don't think it's necessary to redistribute the operations I removed from redacted_cleanup to all of the tests that used it.

Reply:
Thanks, I missed that this is in a cleanup function.